components/script/dom/webglrenderingcontext.rs

```diff
@@ -4090,6 +4092,14 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
         handle_potential_webgl_error!(self, self.validate_ownership(program), return None);
         handle_potential_webgl_error!(self, program.attached_shaders().map(Some), None)
     }
+
+    /// https://immersive-web.github.io/webxr/#dom-webglrenderingcontextbase-makexrcompatible
+    fn MakeXRCompatible(&self) -> Rc<Promise> {
+        // XXXManishearth Fill in with compatibility checks when rust-webxr supports this
```

Comment:
@asajeffrey AIUI magic leap cares about this, but there's no way to get this information from rust-webvr right now.

I'm adding this early because a lot of webxr code seems to want this API to exist.